go.mod

```diff
@@ -3,62 +3,81 @@ module github.com/IBM/ibmcloud-volume-file-vpc
 go 1.18
 
 require (
-	github.com/BurntSushi/toml v0.3.1
 	github.com/IBM-Cloud/ibm-cloud-cli-sdk v0.6.7
-	github.com/IBM/ibm-csi-common v1.0.0-beta17
-	github.com/IBM/ibmcloud-volume-interface v1.0.0-beta10
+	github.com/IBM/ibm-csi-common v1.0.0-beta21
+	github.com/IBM/ibmcloud-volume-interface v1.0.1-beta7.0.20221014111644-b1f895203a78
+	github.com/IBM/secret-common-lib v1.0.4-0.20221017105240-21e98d568724
```

Comment:
Created a release for secret-common-lib too, please update the same.

Reply:
Sure. Will update both secret-common-lib and interface lib once https://github.com/IBM/ibmcloud-volume-interface/pull/25 is merged